screener/models.py

```diff
@@ -400,6 +401,7 @@ def has_benefit(self, name_abbreviated: str):
             "il_ctc": self.has_il_ctc,
             "il_transit_reduced_fare": self.has_il_transit_reduced_fare,
             "il_bap": self.has_il_bap,
+            "il_csfp": self.has_il_csfp,
```

Comment:
We can probably reuse `has_csfp` instead of adding an IL specific field here, yeah?